docs/integrate/debezium/tutorial.md

```
@@ -0,0 +1,277 @@
+For this we need to download the zip file from [kafka-connect-jdbc](https://www.confluent.io/hub/confluentinc/kafka-connect-jdbc) and then run the below replacing `*************` with the appropriate URL:
+
+```bash
+mkdir customdockerimg
+cd customdockerimg
+wget *************/confluentinc-kafka-connect-jdbc-10.6.3.zip
+sudo apt install unzip
+mkdir confluentinc-kafka-connect-jdbc-10.6.3
+cd confluentinc-kafka-connect-jdbc-10.6.3
+unzip -j ../confluentinc-kafka-connect-jdbc-10.6.3.zip
+cd ..
+cat > Dockerfile <<EOF	
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 23

(MD010, no-hard-tabs)